Rakefile

```diff
@@ -5,7 +5,7 @@ RSpec::Core::RakeTask.new(:spec)
 
 desc "Lint Ruby"
 task :lint do
-  sh "bundle exec rubocop --format clang"
```

Comment:
I did always wonder what this was for. Thanks for removing!

Reply:
Yeah - I'm not sure why it ever started, but it has sure spread through our codebase without anyone really understanding (anymore) if it's of value.